ui.apps/src/main/content/jcr_root/apps/saas-aem-module/components/helloworld/helloworld.html

```diff
@@ -19,8 +19,9 @@ <h2 class="cmp-helloworld__title">Hello World Component</h2>
         <p class="cmp-helloworld__item-label">Text property:</p>
         <pre class="cmp-helloworld__item-output" data-cmp-hook-helloworld="property">${properties.text}</pre>
     </div>
-    <div class="cmp-helloworld__item" data-sly-use.model="com.valtech.aemsaas.core.models.HelloWorldModel" data-sly-test="${model.message}">
+    <div class="cmp-helloworld__item" data-sly-use.model="com.valtech.aem.saas.core.models.HelloWorldModel"
```

Comment:
Since the model is deleted, i would delete this component completely

Reply:
thx for the hint :)
it seems that this is why the CI build is failing